main/src/mill/main/RunScript.scala

```diff
@@ -22,94 +22,112 @@ import scala.reflect.ClassTag
   * directly without going through Ammonite's main-method/argument-parsing
   * subsystem
   */
-object RunScript{
-  def runScript(home: Path,
-                wd: Path,
-                path: Path,
-                instantiateInterpreter: => Either[(Res.Failing, Seq[(Path, Long)]), ammonite.interp.Interpreter],
-                scriptArgs: Seq[String],
-                stateCache: Option[Evaluator.State],
-                log: Logger,
-                env : Map[String, String])
-  : (Res[(Evaluator[Any], Seq[PathRef], Either[String, Seq[Js.Value]])], Seq[(Path, Long)]) = {
+object RunScript {
+  def runScript(
+      home: Path,
+      wd: Path,
+      path: Path,
+      instantiateInterpreter: => Either[(Res.Failing, Seq[(Path, Long)]),
+                                        ammonite.interp.Interpreter],
+      scriptArgs: Seq[String],
+      stateCache: Option[Evaluator.State],
+      log: Logger,
+      env: Map[String, String])
+    : (Res[(Evaluator[Any], Seq[PathRef], Either[String, Seq[Js.Value]])],
+       Seq[(Path, Long)]) = {
 
-    val (evalState, interpWatched) = stateCache match{
-      case Some(s) if watchedSigUnchanged(s.watched) => Res.Success(s) -> s.watched
+    val (evalState, interpWatched) = stateCache match {
+      case Some(s) if watchedSigUnchanged(s.watched) =>
+        Res.Success(s) -> s.watched
       case _ =>
-        instantiateInterpreter match{
+        instantiateInterpreter match {
           case Left((res, watched)) => (res, watched)
           case Right(interp) =>
             interp.watch(path)
             val eval =
-              for(rootModule <- evaluateRootModule(wd, path, interp, log))
-              yield Evaluator.State(
-                rootModule,
-                rootModule.getClass.getClassLoader.asInstanceOf[SpecialClassLoader].classpathSignature,
-                mutable.Map.empty[Segments, (Int, Any)],
-                interp.watchedFiles
-              )
+              for (rootModule <- evaluateRootModule(wd, path, interp, log))
+                yield
```

Comment:
I would prefer `for` and `yield` to be vertically aligned, rather than having the `yield` indented